Thunderbird PN: formatting update

litkaemily opened the pull request:
Creating a new pull request since the last one was closed. Removing the bullet points. Still need to figure out how to update the text format and overall general formatting to be consistent with the rest of the notices.

peiying2 commented:
> Creating a new pull request since the last one was closed. Removing the bullet points. Still need to figure out how to update the text format and overall general formatting to be consistent with the rest of the notices.

You can preview the changes you made by clicking on View File

litkaemily commented:
@peiying2 I've updated the effective date and removed the bullet points. Can you see the changes? If so, let's begin the process of pushing to product -- the template change can happen later.
